### PR TITLE
Pull request number four - Desktop version update

### PR DESCRIPTION
**Pull request by ReshyResh**
# Requirements fulfilled succesfully 
 - Made sure no linter errors are present.
 - Used correct Github flow.
 - Used media queries to lay out the desktop version of the portfolio (Figma template n.5, which you can view [here](https://i.imgur.com/hq8W8Jr.jpg).
 - Implemented animated buttons and transitions for better UX.
 - Edited images and backgrounds to fit in the dark theme of the portfolio.
 

# Live Demo
You can see the website in action by clicking *[here](https://reshyresh.github.io/Portfolio/)*

# Screenshots and videos of functionalities
    - Interactable projects sessions, each with their dedicated hover function
    
0cGIeDj

    - Animated buttons 
eSCmXC9

    - Working form 
MQfikEX

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,21 +17,109 @@
...
+          <ul class="project-list">
+            <li class="langbox">HTML/CSS</li>
+            <li class="langbox">JavaScript</li>
+            <li class="langbox">Ruby On Rails</li>
+          </ul>
+          <button class="button-project" type="submit"><span class="button-text">See This Project<i class="fas fa-angle-right"></i></span></button>
+        </section></div>
+
+        <div class="pois second-pois">
+        </div><div class="shape-2"></div>
```
Comment (ZawadiSumba66) [inline index.html:71]: - [ ] In the regular projects(below the project title and the featured project) there should be an image aligned to the left of the page with no margin.
- [ ] There is also a second project that shares the first one's row but is bigger in height (taking also part of the next row also) and looks just like the featured project but the description is above the picture; takes about 25% of the page's width and is aligned completely to the right with no margin in the page.
- [ ] The third project is just a mockup image (like the first one) and is below the first project with no margin on top or left, this one takes 50% of the page's width.
- [ ] Lastly, there's a fourth project below the third one but starts in the center of the page and it takes about 30% of the page's width.
- [ ] Make sure to also add the necessary background designs as well.
All these is similar to the screenshot below.
image

File: index.html [hunk 2]
```diff
@@ -105,17 +193,18 @@ <h2 class="project-title"> Project name goes here </h2>
     </ul>
     </div>
 
-    <div class="main-container">
+    <div class="container-about-section">
+    <div class="container container-middle">
```
Comment (ZawadiSumba66) [inline index.html:197]: - [ ] Let's also adjust the about section part to make a similar design as the one given.
If you look carefully, the skills, frameworks are aligned to the left, and the about part is to the right of it.
- [ ] On the left, there are two rows and two columns, the first row holds Languages and Frameworks respectively, and the second one the Skills.
- [ ] Make sure that you change the about title font to match the one on the template. 👇 
- [ ] The About section also has a white background also.
image

File: index.html [hunk 3]
```diff
@@ -148,27 +237,30 @@ <h3 class="square-text">Skills</h3>
...
+    <section class="container container-bottom">
+      <h1 class="title footer-title">Get Started</h1>
+      <p class="presentation footer-presentation">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
       <form id="contact-form" action="https://formspree.io/f/xqkwdpqy" method="POST">
-        <input name="name" maxlength="30" type="text" class="input-form" required placeholder="Your name">
-        <input name="email" type="email" class="input-form" required placeholder="Your email">
-        <textarea name="message" maxlength="500" class="text-form" required placeholder="Enter your message here.."></textarea>
+        <input name="name" maxlength="30" type="text" class="input-form name-form" required placeholder="Your name">
+        <input name="email" type="email" class="input-form email-form" required placeholder="Your email">
+        <textarea name="message" maxlength="500" class="text-form text-form2" required placeholder="Enter your message here.."></textarea>
```
Comment (ZawadiSumba66) [inline index.html:250]: - [ ] The background color of this section should be `#F9F9FA`. 
- [ ] The name and email input text share the same row.
- [ ] The message input occupies the length of the name and email input.
- [ ] The `Get my CV` button should be placed in the same row as the social media icon links and aligned to the right and the social media icon links to the right.
- [ ] Make sure to also add the other background designs missing.
image

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @ReshyResh 👋 , 

Good job so far!
However, are some issues that you still need to work on to go to the next project but you are almost there! 💪 

## Required Changes ♻️
Please ensure that you replicate the desktop version of Figma design as much as possible by using the media queries. Currently, the design on your desktop version is as similar as your mobile version. Let's fix this 👍 
_Check the comments under the review._

## Optional suggestions
_N/A_
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**